index.html

```diff
@@ -35,5 +35,156 @@ <h2>I'm a Software Developer</h2>
             </ul>
         </div>        
     </section>
+    <section class="works">
+        <div class="card-1">
+            <span>My Recent works</span>
+        </div>
+        <div class="card-2">
+            <img src="/images/gym-image.png" alt="Placeholder">
```

Comment:
- [ ] All the images on your website are currently not displaying. So I think you should add a dot(.) at the beginning f the relative path of all your images url so that they will be visible 

e.g
`<img src="./images/gym-image.png" alt="Placeholder">`